.changeset/eight-poets-count.md

```diff
@@ -0,0 +1,16 @@
+---
+'@leafygreen-ui/tabs': major
```

Comment:
nit, can we separate the major changes from minor and patches? Might make it easier to debug things in the future if we need to

Reply:
I like consolidating into a single release. I think the major changes are fairly trivial, and the test harnesses should help simplify the migration. The minor/patch changes are additive